packages/vite/src/node/server/transformRequest.ts

```diff
@@ -205,6 +205,9 @@ async function loadAndTransform(
         debugLoad?.(`${timeFrom(loadStart)} [fs] ${prettyUrl}`)
       } catch (e) {
         if (e.code !== 'ENOENT') {
+          if (e.code === 'EISDIR') {
+            e.message = `${e.message} ${url}`
```

Comment:
Since we're enhancing a node error message, I think it'll be better if we use `file` so it represents the full file path. Otherwise the user might think we're incorrectly reading/loading the root-relative path as absolute path.
```suggestion
            e.message = `${e.message} ${file}`
```

Reply:
At first, I thought that exposing the absolute path of the file in the error message could help users locate the problematic file more efficiently.
You're right, though, thanks!